ts/components/screens/BaseHeader.tsx

```diff
@@ -48,39 +52,65 @@ const styles = StyleSheet.create({
   body: {
     alignItems: "center"
   },
-  helpButton: {
+  rightButton: {
     padding: 8
   }
 });
 
-const HelpButton: FC<HelpButtonProps> = ({ onShowHelp }) => (
-  <ButtonDefaultOpacity
-    hasFullHitSlop
-    onPress={onShowHelp}
-    transparent={true}
-    accessibilityLabel={I18n.t(
-      "global.accessibility.contextualHelp.open.label"
+const HelpButton: FC<HelpButtonProps> = ({ onShowHelp, dark }) => (
+  <>
+    {itWalletEnabled ? (
+      <View style={styles.rightButton}>
+        <IconButton
+          onPress={onShowHelp}
+          accessibilityLabel={I18n.t(
+            "global.accessibility.contextualHelp.open.label"
+          )}
+          accessibilityHint={I18n.t(
+            "global.accessibility.contextualHelp.open.hint"
+          )}
+          testID="helpButton"
```

Comment:
```suggestion
          testID={"helpButton"}
```

Reply:
We might set a linter rule to enforce this.